src/main/scala/chiseltest/simulator/VcsSimulator.scala

```diff
@@ -173,7 +173,8 @@ private object VcsSimulator extends Simulator {
     "-quiet",
     "-sverilog",
     "-timescale=1ns/1ps",
-    "-debug_pp",
+    "-debug_acc+pp+dmptf",
+    "-debug_region+cell+encrypt",
```

Comment:
Since you have access to the VCS manual, could you add a small comment explaining why these options are needed? I honestly just copied over the `-debug_pp` from our old code with no idea as to why it is needed.

Reply:
Sure, for the original option `debug_pp`, here is the explaination from VCS User Guide
-debug_pp
> It enables read/write access and callbacks to design nets, memory callback, assertion debug, VCS DKI, and VPI routine usage.

For the `debug_acc` option: 
`+pp` means:
> This option enables write capability on registers and variables, callbacks for the entire design, driver capability, and assertion debug capability.

`+dmptf` means:
> This option enables dumping of ports and internal nodes/memories of tasks/functions.

For the `debug_region` option:
`+cell` means:
> Applies debug capabilities to both real cell modules and the ports of real cell modules.

`+encrypt` means:
> Applies debug capabilities to the fully-encrypted instances (modules, programs, packages, and interfaces).

I did not find debug_pp in latest VCS documentation, I believe these should be just equivalent options.

Reply:
Do you know what happens if you just remove these options? Do tests run faster?
We never really access any internal registers or variables via VPI, only the top-level ports.

Reply:
I don't think it is only related VPI access. It is also used for waveform dump. If I turned off this option, the VPD and FSDB waveform dump will crash. I added more comments to explain these option

Reply:
Thanks! The PR will be auto merged once the tests pass.